AnglePanGestureRecognizer/Recognizers/AnglePanGestureRecognizer.swift

```diff
@@ -114,6 +114,31 @@ private struct MovementVector {
 
 public final class AnglePanGestureRecognizer: UIPanGestureRecognizer {
 
+    /// Configuration object to be passed in the constructor
+    public struct Options {
+
+        var allowedAngles: [CGFloat]
+        var unlockedMoveDistance: CGFloat
+
+        public init(allowedAngles: [CGFloat], unlockedMoveDistance: CGFloat) {
+            self.allowedAngles = allowedAngles
+            self.unlockedMoveDistance = unlockedMoveDistance
+        }
+
+    }
+
+    public init(target: Any?,
```

Comment:
yeah, that was the reason I created a `Options` struct (to avoid an initializer with too many args).